streaming/src/test/scala/org/apache/spark/streaming/CheckpointSuite.scala

```diff
@@ -847,6 +847,38 @@ class CheckpointSuite extends TestSuiteBase with DStreamCheckpointTester
     checkpointWriter.stop()
   }
 
+  test("SPARK-28912: Fix MatchError in getCheckpointFiles") {
+    val tempDir = Utils.createTempDir()
```

Comment:
You can use `withTempDir { tempDir =>`

Reply:
Ah @srowen you were more quick with this, +1